### PR TITLE
Feature/813 Add vulnerability assessment with trivy into Makefile

### PR DESCRIPTION
**Pull request by trottomv**

### CODE REVIEW

File: {{cookiecutter.project_dirname}}/Makefile [hunk 1]
```diff
@@ -18,6 +18,23 @@ s =
 rebuild:  ## Rebuild container
 	docker-compose stop $(s) && docker-compose rm -f $(s) && docker-compose up -d --build $(s)
 
+trivy_target = `arg="$(filter-out $@,$(MAKECMDGOALS))" && echo $${arg:-${1}}`
+.PHONY: trivy
+trivy:  ## Run image vulnerabilities scan with 'trivy'
+	docker run --rm \
+	--entrypoint="" \
+	--env CUSTOM_GID=`id -g` \
+	--env CUSTOM_UID=`id -u` \
```
Comment (pauloxnet) [inline {{cookiecutter.project_dirname}}/Makefile:27]:
```suggestion
	--env CUSTOM_GID=`id --group` \
	--env CUSTOM_UID=`id --user` \
```

File: {{cookiecutter.project_dirname}}/scripts/trivy.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    --format template \
+    --output "/app/.trivy/${REPORT_NAME}.html" \
+    --security-checks vuln \
+    --template "@contrib/html.tpl" \
+    ${TARGET_IMAGE}
+time trivy image \
+    --exit-code 0 \
+    --security-checks vuln \
+    ${TARGET_IMAGE}
+chown -R ${CUSTOM_UID}:${CUSTOM_GID} /app/.trivy
```
Comment (pauloxnet) [inline {{cookiecutter.project_dirname}}/scripts/trivy.sh:18]:
```suggestion
chown --recursive ${CUSTOM_UID}:${CUSTOM_GID} /app/.trivy
```

**Review by pauloxnet — CHANGES_REQUESTED** [review]

File: {{cookiecutter.project_dirname}}/Makefile [hunk 3]
```diff
@@ -18,6 +18,23 @@ s =
 rebuild:  ## Rebuild container
 	docker-compose stop $(s) && docker-compose rm -f $(s) && docker-compose up -d --build $(s)
 
+trivy_target = `arg="$(filter-out $@,$(MAKECMDGOALS))" && echo $${arg:-${1}}`
+.PHONY: trivy
+trivy:  ## Run image vulnerabilities scan with 'trivy'
```
Comment (trottomv) [inline {{cookiecutter.project_dirname}}/Makefile:23]:
```suggestion
trivy:  ## Run image vulnerabilities scan with ex. 'make trivy backend'
```

File: {{cookiecutter.project_dirname}}/Makefile [hunk 4]
```diff
@@ -18,6 +18,23 @@ s =
 rebuild:  ## Rebuild container
 	docker-compose stop $(s) && docker-compose rm -f $(s) && docker-compose up -d --build $(s)
 
+trivy_target = `arg="$(filter-out $@,$(MAKECMDGOALS))" && echo $${arg:-${1}}`
+.PHONY: trivy
+trivy:  ## Run image vulnerabilities scan with ex. 'make trivy backend'
```
Comment (trottomv) [inline {{cookiecutter.project_dirname}}/Makefile:23]:
```suggestion
trivy:  ## Run image vulnerabilities scan ex. 'make trivy backend'
```

**Comment by trottomv** [comment]
consider it a spike for now, postponed

**Pull request closed without merging**